src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class FigureSupplier {
+
```

Comment:
remove the first empty line in all classes/interfaces/methods where it is present